src/redux/data.ts

```diff
@@ -895,10 +895,13 @@ const checkForUpdatedNonce = (transactionData: ZerionTransaction[]) => (
   if (transactionData.length) {
     const { accountAddress, network } = getState().settings;
     const txSortedByDescendingNonce = transactionData
-      .filter(
-        ({ address_from }) =>
-          address_from?.toLowerCase() === accountAddress.toLowerCase()
-      )
+      .filter(tx => {
+        const addressFrom = tx?.address_from;
+        return (
+          addressFrom &&
+          addressFrom.toLowerCase() === accountAddress.toLowerCase()
+        );
+      })
       .sort(({ nonce: n1 }, { nonce: n2 }) => (n2 ?? 0) - (n1 ?? 0));
     const [latestTx] = txSortedByDescendingNonce;
     const { address_from, nonce } = latestTx;
```

Comment:
i believe this. one https://sentry.io/organizations/rainbow-me/issues/3332013511/?project=1855565&query=release%3Ame.rainbow%401.6.17%2B31&statsPeriod=14d is coming from here when there's no latestTx object

Reply:
NIIIICE